plenario/models.py

```diff
@@ -464,6 +467,10 @@ def index(cls, geom=None):
 
         return listing
 
+    @classmethod
+    def tablenames(cls):
```

Comment:
the equivalent to MetaTable's `index` method. Exists because ShapeTable's `index` does something else